helix-term/tests/test/commands.rs

```diff
@@ -153,6 +153,17 @@ async fn test_goto_file_impl() -> anyhow::Result<()> {
     )
     .await?;
 
+    // path ends with ","
+    test_key_sequence(
+        &mut AppBuilder::new().with_file(file.path(), None).build()?,
+        Some("i/tmp/test.txt,<esc>bgf"),
```

Comment:
This test seems a bit strange, the cursor is on the `,` when triggering `gf`, meaning a character that is excluded from the output path.